internal/credential/vault/testing_test.go

```diff
@@ -55,6 +50,7 @@ func Test_TestCredentialLibraries(t *testing.T) {
 	}
 }
 
+/*
```

Comment:
Delete instead of comment out since this was moved.

Reply:
Ty!